src/encoded/static/components/radiation.js

```diff
@@ -142,28 +144,58 @@ for (let i = 0; i < ganttData.length; i++) {
         hovertemplate: "%{customdata}<extra></extra>"
 
     }
-  
-    testShape[i] = {
-        x0: ganttData[i].startDate,
-        x1: endDate,
-        y0: yLabels.indexOf(ganttData[i].id) - 0.4,
-        y1: yLabels.indexOf(ganttData[i].id) + 0.4,
-        line: {width: 0}, 
-      type: 'rect', 
-      xref: 'x2', 
-      yref: 'y', 
-      opacity: 1, 
-      fillcolor: '#29A2CC'
+
+    dumbbellData[i] = {
+      x: [ganttData[i].startDate, endDate],
+      y: [yLabels.indexOf(ganttData[i].id), yLabels.indexOf(ganttData[i].id)],
+      mode: 'lines+markers',
+      hoverinfo:'skip',
+      marker: {
+        color: '#29A2CC',
+        size: 15
+      },
+      line: {
+        color: '#29A2CC',
+        width: 15
+      },
+      type: 'scatter',
+      hoverlabel: {
+        bgcolor: '#29A2CC',
+        font: {color: 'white'}
+      },
+      hovertemplate: "Site: " + ganttData[i].id + "<br>Number of lesions: "+ ganttData[i].numberOfSite + dosage + "<br>Start date: "+ganttData[i].startDate +"<br>End date: "+ganttData[i].endDate + "<extra></extra>"
     }
 }
-let date1 = new Date(ganttData[0].startDate);
-let diagnosisDate = new Date(date1.setMonth(date1.getMonth()-1));
-let date2 = new Date(ganttData[ganttData.length - 1].endDate);
-let deceasedDate = new Date(date2.setMonth(date2.getMonth()+1));
-let minX =new Date(date1.setMonth(date1.getMonth()-1));
+
+let diagnosisDate;
+let date1;
+if (this.props.diagnosis_date != "Not available") {
+  diagnosisDate = new Date(this.props.diagnosis_date);
+  date1 = new Date(this.props.diagnosis_date);;
```

Comment:
can set date1 = diagnosisDate since diagnosisDate is already converted
also remove extra ;